packages/@aws-cdk/aws-apprunner-alpha/lib/service.ts

```diff
@@ -1259,4 +1337,48 @@ export class Service extends cdk.Resource implements iam.IGrantable {
       },
     });
   }
+
+  private renderHealthCheckConfiguration(props: HealthCheckConfiguration) {
+    return {
+      healthyThreshold: props.healthyThreshold,
+      interval: props.interval?.toSeconds(),
+      path: props.path,
+      protocol: props.protocol,
+      timeout: props.timeout?.toSeconds(),
+      unhealthyThreshold: props.unhealthyThreshold,
+    };
+  }
+
+  private validateHealthCheckConfiguration(healthCheckConfiguration?: HealthCheckConfiguration) {
+    if (!healthCheckConfiguration) return;
+
+    if (healthCheckConfiguration.path !== undefined) {
+      if (healthCheckConfiguration.protocol !== HealthCheckProtocolType.HTTP) {
+        throw new Error('path is only applicable when you set Protocol to HTTP');
+      }
+      if (healthCheckConfiguration.path.length === 0) {
+        throw new Error('path length must be greater than 0');
+      }
+    }
+    if (healthCheckConfiguration.healthyThreshold !== undefined) {
+      if (healthCheckConfiguration.healthyThreshold < 1 || healthCheckConfiguration.healthyThreshold > 20) {
+        throw new Error(`healthyThreshold must be between 1 and 20, got ${healthCheckConfiguration.healthyThreshold}`);
+      }
+    }
+    if (healthCheckConfiguration.unhealthyThreshold !== undefined) {
+      if (healthCheckConfiguration.unhealthyThreshold < 1 || healthCheckConfiguration.unhealthyThreshold > 20) {
+        throw new Error(`unhealthyThreshold must be between 1 and 20, got ${healthCheckConfiguration.unhealthyThreshold}`);
+      }
+    }
+    if (healthCheckConfiguration.interval !== undefined) {
+      if (healthCheckConfiguration.interval.toSeconds() < 1 || healthCheckConfiguration.interval.toSeconds() > 20) {
+        throw new Error(`interval must be between 1 and 20 seconds, got ${healthCheckConfiguration.interval}`);
```

Comment:
`interval` and `timeout` should be converted with `toSeconds()`

Reply:
OK, I changed!

https://github.com/aws/aws-cdk/pull/27029/commits/629b13113dd5282b369b196417b7c3e443f7f44f